content/en/seachat/getting-started/01-seachat-intro.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
- [ ] Cmd+F the entire document. Find and "appropriately" replace all `bot` to `agent`